WooCommerce/Classes/ViewRelated/Products/AI/ProductSharingMessageGenerationViewModel.swift

```diff
@@ -101,6 +101,10 @@ final class ProductSharingMessageGenerationViewModel: ObservableObject {
     func handleFeedback(_ vote: FeedbackView.Vote) {
         analytics.track(event: .AIFeedback.feedbackSent(source: .productSharingMessage,
                                                         isUseful: vote == .up))
+        if vote == .down {
+            languageIdentifiedUsingAI = nil
+        }
```

Comment:
Similar comment as above.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10177/commits/d48b678b2433243e93080abd6aadee705a5c8f79